### PR TITLE
Hotels and attractions updates with images

### PR DESCRIPTION
**Pull request by coltonpomeroy**

### CODE REVIEW

**Review by kjroelke — DISMISSED** [review]
Solid code

File: src/sections/LocationDetails.astro [hunk 1]
```diff
@@ -91,7 +99,7 @@ const {
 							<NearbyHotel href={hotelsLink} />
```
Comment (kjroelke) [inline src/sections/LocationDetails.astro:99]: h3 should be used in favor of NearbyHotel component once hotels are confirmed

File: src/sections/LocationDetails.astro [hunk 2]
```diff
@@ -44,6 +49,9 @@ const location = {
 		{ title: 'Universal Studios', src: universal },
 		{ title: 'Knotts Berry Farm', src: farm },
 		{ title: 'Disneyland', src: disney },
+		{ title: 'Ontario Mills Outlet', src: mall },
+		{ title: 'K1 Speed — Indoor Go Karts', src: kart },
+		{ title: 'Victoria Gardents', src: vg }
 	],
```
Comment (kjroelke) [inline src/sections/LocationDetails.astro:55]: I believe we're _only_ featuring mall, go karts and victoria gardens. we can remove the other 3

File: src/sections/LocationDetails.astro [hunk 3]
```diff
@@ -22,13 +25,15 @@ const location = {
...
 		},
 		{
 			name: 'Sheraton Fairplex Suites & Conference Center',
 			address: '601 W McKinley Ave, Pomona, CA 91768',
 			rating: 4.4,
 			googleReviews: 1930,
 			image: sheraton,
+			link: 'https://www.marriott.com/en-us/hotels/ontsi-sheraton-fairplex-suites-and-conference-center/overview/'
 		},
 	],
```
Comment (kjroelke) [inline src/sections/LocationDetails.astro:38]: are these the correct hotels from Scott? I think they're the same as when the conference was going to be located at San Dimas, but now that there's a new location idk if these still make sense

**Review by kjroelke — CHANGES_REQUESTED** [review]
got a few things to clear up with scott re: his video + click up card. Also,

- [ ] we need new Rescue a Generation (RAG) logo in _Partners_ Swiper
- [ ] Can you make each slide in the partners swiper a hyperlink to the partners' websites?
- [ ] Need to add a sponsor to the sponsors swiper.

All assets are in clickup card now https://app.clickup.com/t/8689wwv2t

File: src/sections/LocationDetails.astro (hunk 3 above)
Comment (coltonpomeroy) [reply src/sections/LocationDetails.astro:38]: Nope. Let's confirm these are the correct hotels.

File: src/sections/LocationDetails.astro (hunk 2 above)
Comment (coltonpomeroy) [reply src/sections/LocationDetails.astro]: Done and pushed.

**Review by kjroelke — APPROVED** [review]

**Pull request merged**